packages/env/lib/build-docker-compose-config.js

```diff
@@ -43,13 +43,52 @@ module.exports = function buildDockerComposeConfig( config ) {
 		...themeMounts,
 	];
 
-	const testsMounts = [
-		`${
-			config.coreSource ? config.coreSource.testsPath : 'tests-wordpress'
-		}:/var/www/html`,
-		...pluginMounts,
-		...themeMounts,
-	];
+	let testsMounts;
+	if ( config.coreSource ) {
```

Comment:
Hm, actually, we probably should only do the crazy bindings below when `config.coreSource && config.coreSource.type === 'local'`. Non-local core sources can rely on the copy.

Reply:
...or, we could leave it like this adapt `copyWordPress` to only copy the directory plus `wp-config.php`, `wp-config-sample.php` and `wp-content`. That would mean `wp-env start` is faster which is nice.

Reply:
Fixed in c97ab0715a.